genai-perf/genai_perf/inputs/input_constants.py

```diff
@@ -39,14 +39,21 @@ class PromptSource(Enum):
 
 
 class OutputFormat(Enum):
+    ################################################################
+    # Triton backends
+    ################################################################
+    TENSORRTLLM = auto()
```

Comment:
It looks like the logic added [here](https://github.com/triton-inference-server/perf_analyzer/pull/147/files#diff-531d67ceb8ef974e32746d3e8b7b2cd958bac761568f74220ef436b018cacf99R641) expected the options to only list the Triton backends. However, because that was unclear when adding OutputFormats here, it got out of sync.

We might need a better approach long-term as its own ticket, but I did clean-up here to fix the issue for now. 
<img width="1247" alt="image" src="https://github.com/user-attachments/assets/adde4bdf-0909-4ac5-a10f-5869cfda0706">